### ISSUE TITLE
Docs site 404s, seems like its down

### ISSUE THREAD

**Issue by audunmo** [golangci/golangci-lint#4020]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've read the typecheck section of the FAQ (https://golangci-lint.run/usage/faq/#why-do-you-have-typecheck-errors).
- [X] Yes, I've tried with the standalone linter if available (e.g., gocritic, go vet, etc.). (https://golangci-lint.run/usage/linters/)

### Description of the problem

golangci-lint.run 404s, it’s seemingly offline. 

### Version of golangci-lint

<details>

```console
$ golangci-lint --version
# Paste output here
```
  
</details>

### Configuration

<details>

```console
# paste configuration file or CLI flags here
```

</details>

### Go environment

<details>

```console
$ go version && go env
# paste output here
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# paste output here
```
 
</details>

### A minimal reproducible example or link to a public repository

<details>

```go
// add your code here
```
 
</details>

### Validation

- [X] Yes, I've included all information above (version, config, etc.).

**Comment by nilsherzig** [comment]
Yea looks like the build action is running since ~ 80mins https://github.com/golangci/golangci-lint/actions/runs/5890887619/job/15976887669

**Comment by ldez** [comment]
Fixed by https://github.com/golangci/golangci-lint/pull/4014

**Comment by jamietanna** [comment]
FYI even though [the deploy has passed](https://github.com/golangci/golangci-lint/actions/runs/5892325360/job/15981329623) we're still seeing 404s
